app/assets/stylesheets/content/_forms.sass

```diff
@@ -367,30 +362,13 @@ fieldset.form--fieldset
     @extend %form--field-after-container
 
 .form--label
-  @include grid-content(2)
-  @extend %label-style
-  // override max-width set by including grid-content
-  max-width:    100%
-  min-width:    100px
-  flex-grow:    1
-  overflow-x: hidden
-  white-space: normal
-  overflow-y: visible
-  padding:      0 1rem 0 0
+  width: 200px
```

Comment:
You know how I think about fixed widths. Sooner or later this will com back to us. However I did not find any issues because of this, yet. So it is fine for me.

Reply:
Due to the grid layout, I could remove this now.